Sources_v3/StreamChatUI/Generated/L10n.swift

```diff
@@ -1,3 +1,7 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
```

Comment:
I know it sucks this file is generated and we still run formatter on it, but can you please commit only the changes that are relevant. Hopefully we will solve this soon. Thanks